### ISSUE TITLE
Feature: Interaction with the page

### ISSUE THREAD

**Issue by tavareshenrique** [champ3oy/MobileView#7]
Can I interact with the elements?

Make scrolling movements, for example.

n the same way that Chrome and other browsers do, with this little ball with little opacity:

https://github.com/champ3oy/MobileView/assets/27022914/84247839-ab7b-43a7-bcb9-15665d338067

**Comment by champ3oy** [comment]
Yes, all interactions should work as it will on any other browser.

**Comment by tavareshenrique** [comment]
> Yes, all interactions should work as it will on any other browser.

Right! But for me, unfortunately, it doesn't work.

I used the same example that I brought, notice that I can't click and drag to scroll, as if I were using my fingers, the little ball with low opacity doesn't appear.

Do I need to activate it somehow?

https://github.com/champ3oy/MobileView/assets/27022914/8051b3b1-406e-44bf-9edf-ef4c81740239

**Comment by champ3oy** [comment]
Unfortunately, it can't handle touch events yet

**Comment by tavareshenrique** [comment]
Right! I understand! That's why I opened the issue, for a future suggestion. Maybe in the future I can help with that. 😄
Thank you @champ3oy.
